src/keycloak/keycloak_openid.py

```diff
@@ -711,3 +713,31 @@ def register_client(self, token: str, payload: dict):
             URL_CLIENT_REGISTRATION.format(**params_path), data=json.dumps(payload)
         )
         return raise_error_from_response(data_raw, KeycloakPostError)
+
+    def update_client(self, token: str, client_id: str, payload: dict):
```

Comment:
Shouldn't this be in the KeycloakAdmin class instead?

Reply:
This is to update dynamically created clients using a registration access token. Typically for a client previously dynamically created with an initial access token. This allows OpenID client applications to create and update their own keycloak client without having admin access to keycloak. Since dynamic client registration is part of the OpenID specification (https://openid.net/specs/openid-connect-registration-1_0.html), I think this belongs in the OpenID client.